dependencydiff/dependencydiff_test.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencydiff
+
+import (
+	"context"
+	"path"
+	"testing"
+
+	"github.com/ossf/scorecard/v4/checks"
+	"github.com/ossf/scorecard/v4/clients"
+	"github.com/ossf/scorecard/v4/log"
+	"github.com/ossf/scorecard/v4/pkg"
+)
+
+// Test_fetchRawDependencyDiffData is a test function for fetchRawDependencyDiffData.
+func Test_fetchRawDependencyDiffData(t *testing.T) {
+	t.Parallel()
+	//nolint
```

Comment:
those feel more like e2e tests. Maybe move them to the e2e tests instead.

Reply:
Yeah I was also suspicious if these should be a part of the e2e test. Since those functions are called in the main API `GetDependencydiffResult` and the API has already been tested in e2e, maybe I can simply get them removed?